i18n/id.json

```diff
@@ -0,0 +1,265 @@
+{
+  "I18N_SPLASH_PAGE_TITLE": "Oppia: Mengajar, Belajar, Menjelajah",
+  "I18N_SPLASH_TITLE": "Berpikir di luar buku.",
+  "I18N_SPLASH_SUBTITLE": "Oppia mempermudah penciptaan eksplorasi interaktif yang mendidik dan menarik.",
+  "I18N_SPLASH_BROWSE_EXPLORATIONS": "Lihat eksplorasi kami",
+  "I18N_SPLASH_CREATE_EXPLORATION": "Buat eksplorasimu sendiri",
+  "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "Eksplorasi adalah pelajaran interaktif yang memberikan pengalaman lebih dalam dari video atau teks statis, membantu pengguna belajar sambil mempraktekkan.",
+  "I18N_SPLASH_SECOND_EXPLORATION_DESCRIPTION": "Eksplorasi mudah dibuat. Sesuaikan materi dengan mudah berdasarkan feedback individual murid dan tren dari pengalaman semua pelajar anda dari seluruh dunia.",
+  "I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION": "Dengan Oppia, ciptakan dan bagikan eksplorasimu dalam berbagai subjek, hanya imajinasimu batasnya.",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_TITLE": "Kami Membutuhkan JavaScript dalam Browser Anda",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION": "Oppia adalah platform pendidikan gratis dan open-source, penuh dengan aktivitas interaktif bernama 'eksplorasi'. Sayangnya, Oppia membutuhkan JavaScript diaktifkan dalam web browser anda agar dapat berfungsi dengan baik, namun JavaScript web browser anda nonaktif. Jika anda perlu bantuan mengaktifkan JavaScript, <a href=\"<[hrefUrl]>\">klik di sini.</a>",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_THANKS": "Terima kasih.",
+  "I18N_SPLASH_SITE_FEEDBACK": "Feedback Situs",
+  "I18N_LIBRARY_PAGE_TITLE": "Perpustakaan Eksplorasi - Oppia",
+  "I18N_LIBRARY_SEARCH_PLACEHOLDER": "Apa yang ingin anda ketahui?",
+  "I18N_LIBRARY_MAIN_HEADER": "Bayangkan apa yang dapat anda pelajari hari ini...",
+  "I18N_LIBRARY_SUB_HEADER": "Mulai petualangan anda dengan melihat koleksi eksplorasi kami.",
+  "I18N_LIBRARY_ALL_CATEGORIES": "Semua Kategori",
+  "I18N_LIBRARY_ALL_CATEGORIES_SELECTED": "Semua kategori terpilih",
+  "I18N_LIBRARY_CATEGORIES_ALGORITHMS": "Algoritma",
+  "I18N_LIBRARY_CATEGORIES_ARCHITECTURE": "Arsitektur",
+  "I18N_LIBRARY_CATEGORIES_ART": "Seni",
+  "I18N_LIBRARY_CATEGORIES_BIOLOGY": "Biologi",
+  "I18N_LIBRARY_CATEGORIES_BUSINESS": "Bisnis",
+  "I18N_LIBRARY_CATEGORIES_CHEMISTRY": "Kimia",
+  "I18N_LIBRARY_CATEGORIES_CODING": "Koding",
+  "I18N_LIBRARY_CATEGORIES_COMPUTING": "Komputasi",
+  "I18N_LIBRARY_CATEGORIES_ECONOMICS": "Ekonomi",
+  "I18N_LIBRARY_CATEGORIES_EDUCATION": "Pendidikan",
+  "I18N_LIBRARY_CATEGORIES_ENGINEERING": "Teknik",
+  "I18N_LIBRARY_CATEGORIES_ENVIRONMENT": "Lingkungan",
```

Comment:
That would be very useful when more parts of this project is internationalised.
Alright, editing my PR.